### DISCUSSION TITLE
Documentation for "spike_height" seems to be incorrect

### DISCUSSION THREAD

**Discussion by sandromario** [jertel/elastalert2#490] [Q&A] [answered]
Hello,

according to https://elastalert2.readthedocs.io/en/latest/ruletypes.html#rule-configuration-cheat-sheet, the configuration option `spike_height` is an integer. However, according to the schema (https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml), it is a `number`, i.e. a `float`. Which is correct? I would appreciate if it were indeed a float, since I would like to implement a rule of the type "the number of events has risen by 20%". The code looks like passing a float is ok, but could you verify?

Thank you!

**Comment by nsano-rururu** [comment, marked as answer]
I think the numbers were set to allow both integers and floating point numbers.

example 

```yaml
spike_height: 2
```

or


```yaml
spike_height: 2.5
```

[json schema](https://pypi.org/project/jsonschema/) specifications

**Reply by nsano-rururu** [reply]
There were a few patterns in the spike_height test code. It seems that the reason for setting number in the schema settings is to intentionally allow both integers and small numbers. In addition, I searched past issues of the original yelp / elastalert, but most of them are integers. Probably because it is described as int in the document.

https://github.com/jertel/elastalert2/blob/master/tests/rules_test.py#L420

```
def test_spike_terms_query_key_alert_on_new_data():
    rules = {'spike_height': 1.5,
             'timeframe': datetime.timedelta(minutes=10),
             'spike_type': 'both',
             'use_count_query': False,
             'timestamp_field': 'ts',
             'query_key': 'username',
             'use_term_query': True,
             'alert_on_new_data': True}
```

**Reply by sandromario** [reply]
Thanks! So if works with floats too, shall we change it in the documentation?

**Reply by nsano-rururu** [reply]
>Thanks! So if works with floats too, shall we change it in the documentation?

If it works on the float, update it.

**Reply by sandromario** [reply]
Ok, I will if it works.
